plugins/modules/packaging/os/pkgutil.py

```diff
@@ -96,117 +148,129 @@ def run_command(module, cmd, **kwargs):
     return module.run_command(cmd, **kwargs)
 
 
-def package_install(module, state, name, site, update_catalog):
-    cmd = ['pkgutil', '-iy']
+def package_install(module, state, pkgs, site, update_catalog, force):
+    cmd = ['pkgutil']
+    if module.check_mode:
+        cmd.append('-n')
+    cmd.append('-iy')
     if update_catalog:
-        cmd += ['-U']
+        cmd.append('-U')
     if site is not None:
-        cmd += ['-t', site]
-    if state == 'latest':
-        cmd += ['-f']
-    cmd.append(name)
-    (rc, out, err) = run_command(module, cmd)
-    return (rc, out, err)
+        cmd.extend('-t', site)
+    if force:
+        cmd.append('-f')
+    cmd.extend(pkgs)
+    return run_command(module, cmd)
 
 
-def package_upgrade(module, name, site, update_catalog):
-    cmd = ['pkgutil', '-ufy']
+def package_upgrade(module, pkgs, site, update_catalog, force):
+    cmd = ['pkgutil']
+    if module.check_mode:
+        cmd.append('-n')
+    cmd.append('-uy')
     if update_catalog:
-        cmd += ['-U']
+        cmd.append('-U')
     if site is not None:
-        cmd += ['-t', site]
-    cmd.append(name)
-    (rc, out, err) = run_command(module, cmd)
-    return (rc, out, err)
+        cmd.extend('-t', site)
+    if force:
+        cmd.append('-f')
+    cmd += pkgs
+    return run_command(module, cmd)
 
 
-def package_uninstall(module, name):
-    cmd = ['pkgutil', '-ry', name]
-    (rc, out, err) = run_command(module, cmd)
-    return (rc, out, err)
+def package_uninstall(module, pkgs):
+    cmd = ['pkgutil']
+    if module.check_mode:
+        cmd.append('-n')
+    cmd.append('-ry')
+    cmd.extend(pkgs)
+    return run_command(module, cmd)
 
 
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            name=dict(required=True),
-            state=dict(required=True, choices=['present', 'absent', 'latest']),
-            site=dict(default=None),
-            update_catalog=dict(required=False, default=False, type='bool'),
+            name=dict(type='list', required=True, aliases=['pkg']),
```

Comment:
```suggestion
            name=dict(type='list', elements='str', required=True, aliases=['pkg']),
```
Why do you add an alias here?

Reply:
I'm not the original author of this, but a `pkg` alias seems reasonable to me for consistency with, e.g., the `yum` module.